SharePointFramework/PortfolioWebParts/src/components/ProjectTimeline/DetailsCallout/index.tsx

```diff
@@ -11,28 +11,51 @@ export interface IDetailsCalloutProps {
 
 // tslint:disable-next-line: naming-convention
 export const DetailsCallout = ({ item, onDismiss }: IDetailsCalloutProps) => {
+  console.log(item);
+
   return (
     <Callout
       className={styles.detailsCallout}
       gapSpace={10}
       target={item.element}
       onDismiss={onDismiss}
       setInitialFocus={true}>
-      <p>
+      <p hidden={item.data.type !== 'Milepæl'}>
+        <b>Milepæl:</b>{' '}
```

Comment:
You're adding these as resources/strings, right? We don't want to add even more hardcoded strings 🚀

Reply:
Of course ;)

Reply:
Better safe than sorry! Resolving convo!